redux-ui/redux-ui.d.ts

```diff
@@ -0,0 +1,50 @@
+/// <reference path="../redux/redux.d.ts"/>
```

Comment:
the header content is missing. see http://definitelytyped.org/guides/contributing.html#content